server/src/main/java/org/elasticsearch/common/network/CloseableChannel.java

```diff
@@ -83,19 +83,15 @@
      * @param blocking indicates if we should block on channel close
      */
     static <C extends CloseableChannel> void closeChannels(List<C> channels, boolean blocking) {
+        IOUtils.closeAndConvertToRuntimeExceptions(channels);
```

Comment:
wouldnt' it be simpler to do:

```Java
try {
  IOUtils.close(channels);
} catch (IOException e) {
  throw new UncheckedIOException(e);
}
```

???

Reply:
Sure. I’ll pull out the utils part and do that.